plugins/ipam/commands/commands_test.go

```diff
@@ -70,7 +71,6 @@ func TestGetSpecificIPV4HappyPath(t *testing.T) {
 
 	gomock.InOrder(
 		allocator.EXPECT().Assign(gomock.Any(), gomock.Any()).Return(nil),
-		allocator.EXPECT().SetLastKnownIP(net.ParseIP("10.0.0.3")),
```

Comment:
https://github.com/aws/amazon-ecs-cni-plugins/blob/55b2ae77ee0bf22321b14f2d4ebbcc04f77322e1/plugins/ipam/commands/commands.go#L160-L168

removed this as the `else` statement is not touched.  It has a call to `getIPV4AddressFromDB` which calls `SetLastKnownIP`